MovingBlocks/Terasology
# fix: restore version info

Fixes #4595 

### How to test

On the main menu, make sure it shows _some_ kind of version identifier (e.g. `Alpha-20`) instead of only "custom build."
